crash-handler-process/platforms/process-win.cpp

```diff
@@ -64,14 +83,83 @@ bool Process_WIN::isCritical(void) {
 	return critical;
 }
 
+void Process_WIN::startMemoryDumpMonitoring(void) {
+	if (memorydump && memorydump->joinable()) {
+		return;
+	}
+
+	mds = OpenEvent(EVENT_ALL_ACCESS, FALSE, L"Global\\OBSMEMORYDUMPEVENT");
+	if (!mds || mds == INVALID_HANDLE_VALUE) {
+		log_info << "Failed to open event for memory dump " << GetLastError()<< std::endl;
+		return;
+	}
+
+	PSECURITY_DESCRIPTOR securityDescriptor = (PSECURITY_DESCRIPTOR) LocalAlloc(LPTR, SECURITY_DESCRIPTOR_MIN_LENGTH);
+	if (InitializeSecurityDescriptor(securityDescriptor, SECURITY_DESCRIPTOR_REVISION)) {
+		if (SetSecurityDescriptorDacl( securityDescriptor, TRUE, NULL, FALSE)) {
+			SECURITY_ATTRIBUTES eventSecurityAttr = {0};
+			eventSecurityAttr.nLength = sizeof(SECURITY_ATTRIBUTES);
+			eventSecurityAttr.lpSecurityDescriptor = securityDescriptor;
+			eventSecurityAttr.bInheritHandle = FALSE;
+
+			mdf = CreateEvent( &eventSecurityAttr, TRUE, FALSE, L"Global\\OBSMEMORYDUMPFINISHEDEVENT");
```

Comment:
Not saying it's an issue but why both osn and the crash handler have to Create the event? Only one of them doing it is not enough?

Reply:
There is two events. One for signaling when crash happens and we could to start saving memory to a dump. And another when saving is finished and crashed process could be closed. 

If everything going correctly each process creates one and opens another.